bindata/assets/kube-controller-manager/svc-cluster-policy-controller.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-policy-controller
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: serving-cert-cpc
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: https
+  labels:
+    prometheus: "cluster-policy-controller"
+spec:
+  selector:
+    kube-controller-manager: "true"
+  ports:
+  - name: https
+    port: 443
```

Comment:
Do we really need to have the service listening on a privileged port? It might be better to use the same port as the one used by the container wdyt?

Reply:
@dgrisonnet I have switched to the respective container ports. Although I am not sure why 443 port is problematic for services, can you elaborate on that a bit?

Reply:
I am not aware of any with services, but I tend to avoid using privileged ports as much as possible and here I don't think there is a real reason for that.

Reply:
ok, thx